integrations/twilio/client.go

```diff
@@ -40,6 +44,49 @@ func New(vars sdkservices.Vars) sdkservices.Integration {
 				sdkmodule.WithArgs("to", "from_number?", "messaging_service_sid?", "body?", "media_url?", "content_sid?"),
 			),
 		),
+		connStatus(i),
 		sdkintegrations.WithConnectionConfigFromVars(vars),
 	)
 }
+
+// connStatus is an optional connection status check provided by the
+// integration with AutoKitteh. The possible results are "init required"
+// (indicating the connection is not yet usable), "using X" (indicating
+// one of multiple available authentication methods is in use), or
+// "initialized" when only one authentication method is available.
+func connStatus(i *integration) sdkintegrations.OptFn {
+	return sdkintegrations.WithConnectionStatus(func(ctx context.Context, cid sdktypes.ConnectionID) (sdktypes.Status, error) {
+		if !cid.IsValid() {
+			return sdktypes.NewStatus(sdktypes.StatusCodeWarning, "init required"), nil
+		}
+
+		vs, err := i.vars.Get(ctx, sdktypes.NewVarScopeID(cid))
+		if err != nil {
+			return sdktypes.InvalidStatus, err
+		}
+
+		var decodedVars webhooks.Vars
+		vs.Decode(&decodedVars)
+
+		s, err := sdktypes.ParseSymbol("AuthType")
```

Comment:
i would put this in the global vars, using NewSymbol. this is effectively a constant.

Reply:
Also, why `AuthType` and not `authType`?

Reply:
@itayd this is happening through EncodeVars. I tried removing EncodeVars, but it breaks a lot of the starlark stuff so I decided not to touch it. I could do a separate vars, but I thought it looked cleaner this way.

Reply:
@daabr because
> Note that EncodeVars uses the field's name, i.e. AuthType

**i.e.**

`auth.go`:
```
type Vars struct {
        // line below
	AuthType   string
	AccountSID string
	Username   string `vars:"secret"`
	Password   string `vars:"secret"`
}
```